### PR TITLE
LO-5765: Data Lake Table Schema Commands

### PR DESCRIPTION
**Pull request by mjtieman**
Added commands to fetch the schemas of each table in the data lake, for a project, and schema of a single table.

### CODE REVIEW

File: lib/cmds/data_lake_cmds/get-schema.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+exports.command = 'get-schema <projectId>';
+exports.desc = 'Fetch a single query execution.';
+exports.builder = yargs => {
+  yargs.positional('projectId', {
+    describe: 'The ID of the project to fetch the schema from.',
+    type: 'string'
+  }).option('table', {
+    alias: 't',
+    type: 'string',
+    describe: 'Name of the table to fetch the schema of.',
```
Comment (mschroering) [inline lib/cmds/data_lake_cmds/get-schema.js]: Are the table names dynamic?  If not, you can add an enumerated set to aid a user.

Comment (mjtieman) [reply lib/cmds/data_lake_cmds/get-schema.js]: The tables are dynamic in-that a project will only have tables for which there is data, e.g. a project that only has patient data will not have the `variant` table.

Comment (mschroering) [reply lib/cmds/data_lake_cmds/get-schema.js]: Yeah, so would something like this help:

``` 
choices: ['variant', 'patient', 'gene', ...],
describe: 'Name of the table to fetch the schema of (The possible list of choices are provided. Note that not every project will have every table)',
```

Comment (mjtieman) [reply lib/cmds/data_lake_cmds/get-schema.js]: Done

**Pull request merged**